README.md

```diff
@@ -54,15 +54,19 @@ pushd kythe-verification; stack install && ./test.sh; popd
 
 # Demo
 
-To index a few packages and serve the index, run:
+To index a few packages, run:
 
-```
+```bash
 ./build-stack.sh /tmp/logs mtlparse cpu
 ```
 
-The script temporarily replaces the system GHC with
-`wrappers/stack-docker/fake-stack/ghc` script, does the indexing and serves the
-built index at `localhost:8080`.
+The script installs a post hook to `stack path --compiler-exe`, does the indexing when `ghc --make` is detected to build a package. You can run `build-stack.sh` multiple times.
```

Comment:
It's called once.

I guess it is related to `-dynamic-too` be (seems to be supported since some very old release) coming the default (one build for both vanilla (.a) and dynamic ways (.so)).
` == args: --make -fbuilding-cabal-package -O -static -dynamic-too -dynosuf dyn_o -dynhisuf dyn_hi -outputdir .stack-work/dist/x86_64-linux/Cabal-1.24.2.0/build -odir .stack-work/dist/x86_64-linux/Cabal-1.24.2.0/build -hidir .stack-work/dist/x86_64-linux/Cabal-1.24.2.0/build -stubdir .stack-work/dist/x86_64-linux/Cabal-1.24.2.0/build -i -i.stack-work/dist/x86_64-linux/Cabal-1.24.2.0/build -i. -i.stack-work/dist/x86_64-linux/Cabal-1.24.2.0/build/autogen -I.stack-work/dist/x86_64-linux/Cabal-1.24.2.0/build/autogen -I.stack-work/dist/x86_64-linux/Cabal-1.24.2.0/build -optP-include -optP.stack-work/dist/x86_64-linux/Cabal-1.24.2.0/build/autogen/cabal_macros.h -this-unit-id mtl-2.2.1-BLKBelFsPB3BoFeSWSOYj6 -hide-all-packages -no-user-package-db -package-db /home/ray/.stack/snapshots/x86_64-linux/lts-8.17/8.0.2/pkgdb -package-db .stack-work/dist/x86_64-linux/Cabal-1.24.2.0/package.conf.inplace -package-id base-4.9.1.0 -package-id transformers-0.5.2.0 -XHaskell98 -XMultiParamTypeClasses -XFunctionalDependencies -XFlexibleInstances Control.Monad.Cont Control.Monad.Cont.Class Control.Monad.Error Control.Monad.Error.Class Control.Monad.Except Control.Monad.Identity Control.Monad.List Control.Monad.RWS Control.Monad.RWS.Class Control.Monad.RWS.Lazy Control.Monad.RWS.Strict Control.Monad.Reader Control.Monad.Reader.Class Control.Monad.State Control.Monad.State.Class Control.Monad.State.Lazy Control.Monad.State.Strict Control.Monad.Trans Control.Monad.Writer Control.Monad.Writer.Class Control.Monad.Writer.Lazy Control.Monad.Writer.Strict -Wall -fno-warn-unused-imports -fno-warn-warnings-deprecations -ddump-hi -ddump-to-file`